ghmarkdown/ghmarkdown.py

```diff
@@ -171,10 +177,20 @@ def main():
                         help='silences server output and rate information')
     parser.add_argument('--serve', '-s', action='store_true',
                         help='locally serve parsed markdown')
-    parser.add_argument('--port', '-p', metavar='PORT')
+    parser.add_argument('--auto', '-a', action='store_true',
+                        help='automatically refreshes the server\'s html (default=2s)')
+    parser.add_argument('--setauto', '-sa', metavar='SEC',
```

Comment:
Use `--rate` and `-r` instead, because `-sa` is interpreted by the parser as `-s -a`.